stencil.config.ts

```diff
@@ -26,8 +26,8 @@ export const config: Config = {
   },
   outputTargets: [
     reactOutputTarget({
-      componentCorePackage: '../../dist/types',
-      proxiesFile: './generated/react-bindings/components.ts',
+      componentCorePackage: '../dist/types',
```

Comment:
What's this relative to? (What happens if it's wrong?)

Reply:
It's relative to the proxies file.

![image](https://user-images.githubusercontent.com/2008881/129774096-e8b1eba7-e999-49c1-a668-e18df2ca6770.png)

If it's wrong then we get an error: https://github.com/department-of-veterans-affairs/component-library/pull/131#discussion_r684465784